emuDCS/PeripheralApps/src/common/EmuPeripheralCrateConfig.cc

```diff
@@ -5055,68 +5084,88 @@ void EmuPeripheralCrateConfig::ChamberTests(xgi::Input * in, xgi::Output * out )
+  *out << cgicc::td().set("ALIGN", "left") << "Half-Strip" << cgicc::td() << std::endl;
+  //
+  *out << cgicc::tr();
+  //
+  *out << cgicc::br(); 
+  //
+  *out << cgicc::tr();
+  
+  *out << cgicc::td().set("ALIGN", "left") << std::endl;
+  *out << cgicc::select().set("name", "time_delay") << std::endl;
+  *out << cgicc::option().set("value", toolbox::toString("%d", -1)) << "Scan" << cgicc::option() << std::endl;
+  for(int i=0; i<25; ++i) *out << cgicc::option().set("value", toolbox::toString("%d", i)) << i << cgicc::option() << std::endl;
+  *out << cgicc::select() << std::endl;
+  *out << cgicc::td();
+  *out << cgicc::td().set("ALIGN", "left") << std::endl;
+  *out << cgicc::select().set("name", "cfeb_phase") << std::endl;
+  *out << cgicc::option().set("Value", toolbox::toString("%d", 32)) << "XML Inherited" << cgicc::option() << std::endl;
+  for(int i=0; i<32; ++i) *out << cgicc::option().set("value", toolbox::toString("%d", i)) << i << cgicc::option() << std::endl;
+  *out << cgicc::option().set("value", toolbox::toString("%d", -1)) << "Scan" << cgicc::option() << std::endl;
+  *out << cgicc::select() << std::endl;
+  *out << cgicc::td();
+  *out << cgicc::td().set("ALIGN", "left") << std::endl;
+  *out << cgicc::select().set("name", "cfeb_num") << std::endl;
+  *out << cgicc::option().set("value", toolbox::toString("%d", -1)) << "Scan" << cgicc::option() << std::endl;
+  for(int i=0; i<thisDMB->cfebs_.size(); ++i) *out << cgicc::option().set("value", toolbox::toString("%d", i)) << i << cgicc::option() << std::endl;
+  *out << cgicc::select() << std::endl;
+  *out << cgicc::td();
+  *out << cgicc::td().set("ALIGN", "left") << std::endl;
+  *out << cgicc::select().set("name", "pattern") << std::endl;
+  for(int i=0xa; i>0x1; --i) *out << cgicc::option().set("value", toolbox::toString("%d", i)) << toolbox::toString("0x%x", i) << cgicc::option() << std::endl;
+  *out << cgicc::select() << std::endl;
+  *out << cgicc::td();
+  *out << cgicc::td().set("ALIGN", "left") << std::endl;
+  *out << cgicc::select().set("name", "halfstrip") << std::endl;
+  *out << cgicc::option().set("value", toolbox::toString("%d", -1)) << "Random" << cgicc::option() << std::endl;
+  for(int i=0; i<32; ++i) *out << cgicc::option().set("value", toolbox::toString("%d", i)) << i << cgicc::option() << std::endl;
+  *out << cgicc::select() << std::endl;
+  *out << cgicc::td();
+  *out << cgicc::td();
+  *out << cgicc::input().set("type","submit").set("value", "CFEB RX Delay Scan").set("style", "color:blue") << std::endl;
+  *out << cgicc::td();
+  sprintf(buf,"%d",tmb);
+  *out << cgicc::input().set("type","hidden").set("value",buf).set("name","tmb");
+  sprintf(buf,"%d",dmb);
+  *out << cgicc::input().set("type","hidden").set("value",buf).set("name","dmb");
+  *out << cgicc::form() << std::endl ;
+  //
+  *out << cgicc::table();
+  //
+  *out << cgicc::tr();
+  //
+  *out << cgicc::tr();
+  //
+  *out << cgicc::td();
   //
   for(int i=0;i<5;i++) {
```

Comment:
ChamberUtilities.h:  ``` inline int  GetCFEBrxPhaseTest(int CFEB) { return CFEBrxPhase_[CFEB] ; } ```
the CFEBrxPhase_ is size 7, makes sense to print more if it's OTMB
... fixing it